### DISCUSSION TITLE
New Library arrived: ZoneTree.FullTextSearch

### DISCUSSION THREAD

**Discussion by koculu** [koculu/ZoneTree#89] [Announcements]
# 🚀 Introducing ZoneTree.FullTextSearch: Powerful Full-Text Search for .NET

ZoneTree.FullTextSearch is now available, adding robust full-text search capabilities to .NET applications. Built on the ZoneTree engine, it offers fast and efficient indexing and querying for large datasets.

### Key Features:
- **High Performance**: Quick indexing and search, even with large volumes of data.
- **Advanced Queries**: Supports complex searches with Boolean operators, facets, and more.
- **Customizable**: Easily integrates custom tokenizers, stemmers, and normalizers.
- **Scalable**: Designed to handle large datasets, with in-memory caching for faster queries.

For those who find ZoneTree complex, exploring the ZoneTree.FullTextSearch code is a great way to see how ZoneTree can be utilized effectively in real-world scenarios. It serves as a practical example to help you navigate and understand the powerful features of ZoneTree.

For more details, visit the [ZoneTree.FullTextSearch GitHub Repository](https://github.com/koculu/ZoneTree.FullTextSearch).

Feedback and contributions are welcome via GitHub.

**Comment by harrison314** [comment]
.NET definitely needed something like this.

**Comment by Frooxius** [comment]
Does it support fuzzy matching? I couldn't find any mention of this.

**Reply by koculu** [reply]
No, it doesn't support fuzzy search yet. Actually if something is not in the docs or code, it means it is not supported.
